app/scripts/lib/storage.js

```diff
@@ -24,7 +24,7 @@ define([
     var item;
     try {
       item = JSON.parse(this._backend.getItem(fullKey(key)));
-    } catch (e) {
+    } catch (e) {//eslint-disable-line no-empty
```

Comment:
JSCS doesn't seem to complain about anything to be honest, but I guess that might be cause we haven't set requireSpacesInsideObjectBrackets to true.